src/prashantCJune20/SumofIntinString.java

```diff
@@ -0,0 +1,19 @@
+package prashantCJune20;
+
+public class SumofIntinString {
+	
+	void addDigits(String input) {
+		int addition = 0;
+		for(int index = 0;index<input.length();index++) {
```

Comment:
index of the  character isn't necessary for the execution you can simply just use `forEach` loop